packages/tools/examples/crossHairs/index.ts

```diff
@@ -136,7 +135,50 @@ function getReferenceLineSlabThicknessControlsOn(viewportId) {
   return index !== -1;
 }
 
-// ============================= //
+const blendModeOptions = {
+  MIP: 'Maximum Intensity Projection',
+  MINIP: 'Minimum Intensity Projection',
+  AIP: 'Average Intensity Projection',
+};
+
+addDropdownToToolbar({
+  options: {
+    values: [
+      'Maximum Intensity Projection',
+      'Minimum Intensity Projection',
+      'Average Intensity Projection',
+    ],
+    defaultValue: 'Maximum Intensity Projection',
+  },
+  onSelectedValueChange: (selectedValue) => {
```

Comment:
Sent you a video @sedghi . Because the blend mode only changes when you manipulate the lines that corresponds to a viewport, you can end up with a weird combination of different blendmodes. I think you just need to set all actors to that blendmode and re-render on your swap button.